spec/collections_publisher/archiving_child_topic_spec.rb

```diff
@@ -0,0 +1,75 @@
+feature "Archiving a child topic on Collections Publisher", collections_publisher: true do
+  include CollectionsPublisherHelpers
+
+  let(:parent_title) { title_with_timestamp }
+  let(:parent_slug) { slug_with_timestamp }
+
+  let(:child_slug) { slug_with_timestamp }
+  let(:child_title) { title_with_timestamp }
+  let(:link) { ["/topic", parent_slug, child_slug].join('/') }
+
+  scenario "Archiving a child topic" do
+    given_there_is_a_published_child_topic
+    and_i_archive_it
+    then_i_am_redirected_to_the_parent_on_gov_uk
```

Comment:
I think this should be rephrased slightly as the story is more than trying to visit the child results in a redirect, whereas this sounds like process of archiving it self sends you to gov.uk